cogs/pokemon.py

```diff
@@ -949,42 +949,57 @@ async def get_page(source, menu, pidx):
     @checks.has_started()
     @commands.guild_only()
     @commands.command(rest_is_raw=True)
-    async def evolve(self, ctx, *, pokemon: converters.PokemonConverter):
+    async def evolve(self, ctx, args: commands.Greedy[converters.PokemonConverter]):
         """Evolve a pokémon if it has reached the target level."""
 
-        if pokemon is None:
+        if not all(pokemon is not None for pokemon in args):
             return await ctx.send("Couldn't find that pokémon!")
```

Comment:
Can we specify which Pokémon here?

Reply:
Not entirely sure how to add this seeing as `pokemon` is None here, and the `args: commands.Greedy` doesn't store the initial unconverted argument